packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -2977,18 +3101,21 @@ Widget _navBarHeroFlightShuttleBuilder(
 }
 
 class _NavigationBarSearchField extends StatelessWidget implements PreferredSizeWidget {
-  const _NavigationBarSearchField();
+  const _NavigationBarSearchField({required this.searchKey});
 
   static const double padding = 8.0;
   static const double searchFieldHeight = 35.0;
+  final GlobalKey searchKey;
 
   @override
   Widget build(BuildContext context) {
-    return const Padding(
-      padding: EdgeInsets.symmetric(horizontal: padding, vertical: padding),
-      child: SizedBox(
-        height: searchFieldHeight,
-        child: CupertinoSearchTextField()
+    return IgnorePointer(
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: padding, vertical: padding),
+        child: SizedBox(
+          height: searchFieldHeight,
+          child: CupertinoSearchTextField(key: searchKey),
```

Comment:
Should we expose this `CupertinoSearchTextField` as a property in the `.search` constructor? I wonder if users might want more control over it.